### PR TITLE
Feat/crud meal

### PR DESCRIPTION
**Pull request by JaelynSJP**
- CRUD meal 
- added photo upload

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
   root to: 'pages#home'
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
-end
+  resources :restaurants do 
+    resources :meals
```
Comment (yuanminglee) [inline config/routes.rb:9]: Select only the actions needed.(collection routes)

member routes like destroy will be outside restaurants

File: app/controllers/meals_controller.rb [hunk 2]
```diff
@@ -1,3 +1,50 @@
 class MealsController < ApplicationController
-    def 
+  def index
+    restaurant = Restaurant.find(params[:restaurant_id])
```
Comment (yuanminglee) [inline app/controllers/meals_controller.rb]: Make it an instance variable, should be @restaurant

File: config/routes.rb (hunk 1 above)
Comment (JaelynSJP) [reply config/routes.rb:9]: but to destroy the meal, the chef can only destroy his own restaurant meal right?
so to destroy a meal need the restaurant id as shown in the path?
destroy path

File: app/controllers/meals_controller.rb [hunk 3]
```diff
@@ -1,50 +1,54 @@
 class MealsController < ApplicationController
+  before_action :find_meal, only: [:show, :edit, :update, :destroy]
+  before_action :find_restaurant, only: [:index, :show, :new, :create, :edit, :update, :destroy]
+
   def index
-    restaurant = Restaurant.find(params[:restaurant_id])
-    @meals = Meal.where(restaurant_id: restaurant)
+    @meals = Meal.where(restaurant_id: @restaurant)
```
Comment (yuanminglee) [inline app/controllers/meals_controller.rb:6]: Can just be @restaurant.meals

**Pull request merged**